### DISCUSSION TITLE
[@types/react] should onKeyPress be marked as deprecated?

### DISCUSSION THREAD

**Discussion by ShuiRuTian** [DefinitelyTyped/DefinitelyTyped#58157] [Issues with a @types package] [answered]
https://developer.mozilla.org/en-US/docs/Web/API/Element/keypress_event
keypress event has been deprecated for a long time. Should react mark “onKeyPress” as `deprecated`?

[a comment by an automated account is not shown]

**Comment by eps1lon** [comment, marked as answer]
Note that DOM events are not the same as React events. Deprecating a React event is decided by the React core team. It's likely that these events are also not working in browsers that stopped supporting them but that isn't that clear.

I don't see that need to mark them as deprecated since that may cause people to think that React deprecated these events not the platform.
